common/common_structs/src/locked_token_types.rs

```diff
@@ -149,6 +184,44 @@ pub struct LockedAssetTokenAttributesEx<M: ManagedTypeApi> {
     pub is_merged: bool,
 }
 
+#[derive(
+    ManagedVecItem,
+    TopEncode,
+    TopDecode,
+    NestedEncode,
+    NestedDecode,
+    TypeAbi,
+    Clone,
+    PartialEq,
+    Debug,
+)]
+pub struct InitialLockedAssetTokenAttributesEx<M: ManagedTypeApi> {
```

Comment:
The "Ex" at the end stands for "Extended", which this one is clearly not. I believe there is a LockedAssetAttributes type (without the "Ex"). If so, create an alias for it and use that. Either way, the current name is wrong

Reply:
Done.